### ISSUE TITLE
System Crashing consistly 

### ISSUE THREAD

**Issue by CaptainAzriel** [StrangeLoopGames/EcoIssues#19861]
Providing both server crashes and client crashes
[log_201227102703.log](https://github.com/StrangeLoopGames/EcoIssues/files/5747126/log_201227102703.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5747129/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/5747130/Player-prev.log)

**Comment by CaptainAzriel** [comment]
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5747256/Player.log)

**Comment by IcedForge** [comment]
@CaptainAzriel 
Thank you for reporting, terribly sorry for the late reply.
If this is still happening could you screenshot your F2 debug window ingame and the gpu page in task manager (with the game running )

**Comment by CaptainAzriel** [comment]
Yeah still happening i will do it tonight and email you back, how do I
access the f2 bug window I see it appear when it crashes

**Comment by IcedForge** [comment]
Screencaps of CPU/GPU and ingame texture usage.

![Screenshot_52](https://user-images.githubusercontent.com/24981362/104575660-919c1180-5657-11eb-8ee5-281a464dbd24.jpg)


![Screenshot_53](https://user-images.githubusercontent.com/24981362/104575668-94970200-5657-11eb-98fb-4fadd968affa.jpg)

![Screenshot_54](https://user-images.githubusercontent.com/24981362/104575676-9791f280-5657-11eb-87f9-8a6668fe7012.jpg)
